src/bundle/partial.ts

```diff
@@ -27,49 +43,65 @@ export async function prepareBundlePartialFiles(path: string, snag: LogSnag, org
   return manifest
 }
 
-export async function uploadPartial(apikey: string, manifest: manifestType, path: string, options: Options, appId: string, name: string) {
-  const spinner = spinnerC()
-  spinner.start('Preparing partial update')
-  const uploadResponse: uploadUrlsType[] = await manifestUploadUrls(apikey, appId, name, manifest)
+export async function uploadPartial(apikey: string, manifest: manifestType, path: string, options: any, config: CapacitorConfig, appId: string, name: string) {
+  try {
+    const baseUrl = config?.plugins?.CapacitorUpdater?.cloudflareBaseUrl ?? defaultApiHost
+    const url = new URL(`${baseUrl}/private/partial_upload/v1`)
 
-  if (uploadResponse.length === 0 || uploadResponse.length !== manifest.length) {
-    log.error(`Cannot upload manifest, please try again later`)
-    spinner.stop('Partial update failed')
-    return []
-  }
-  spinner.message('Uploading partial update')
-  for (const [index, manifestEntry] of uploadResponse.entries()) {
-    const finalFilePath = `${path}/${manifestEntry.path}`
-    spinner.message(`Uploading partial update ${index + 1}/${uploadResponse.length}`)
```

Comment:
add back the spinner

Reply:
There is a spinner, just look in the upload code directly